spring-boot-examples/consumer-app/src/test/java/io/dapr/springboot/examples/consumer/DaprTestContainersConfig.java

```diff
@@ -36,31 +36,7 @@ public class DaprTestContainersConfig {
 
   @Bean
   public Network getDaprNetwork() {
-    Network defaultDaprNetwork = new Network() {
-      @Override
-      public String getId() {
-        return "dapr-network";
-      }
-
-      @Override
-      public void close() {
-
-      }
-
-      @Override
-      public Statement apply(Statement base, Description description) {
-        return null;
-      }
-    };
-
-    List<com.github.dockerjava.api.model.Network> networks = DockerClientFactory.instance().client().listNetworksCmd()
-            .withNameFilter("dapr-network").exec();
-    if (networks.isEmpty()) {
-      Network.builder().createNetworkCmdModifier(cmd -> cmd.withName("dapr-network")).build().getId();
-      return defaultDaprNetwork;
-    } else {
-      return defaultDaprNetwork;
-    }
+    return Network.newNetwork();
```

Comment:
@siri-varma this would have been so much easier.. but we cannot do that if we want to run both applications locally to reuse the network.